src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+
+## How do light clients work? {#how-do-light-clients-work}
+
+When Ethereum started using a proof-of-stake based consensus mechanism, new infrastructure was introduced specifically to support light clients. The way it works is by randomly selecting a subset of 512 validators every 1.1 days to act as a "sync-committee". The sync-committee signs the header of recent blocks. Each block header contains the the aggregated signature of the validators in the sync-committee and a "bitfield" that shows which validators signed and which did not. Each header also includes a list of validators expected to participate in signing the next block. This means a light client can quickly see that the sync-committee has signed off on the data they receive, and they can also check that the sync-committee is the genuine one by comparing the one they receive from the one they were told to expect in the previous block. In this way, the light client can keep updating it's knowledge of the latest Ethereum block without actually downloading the block itself, just the header which contains summary information.
+
+On the execution layer there is no single specification for a light execution client. The scope of a light execution client can vary from a "light mode" of a full execution client that has all the EVM and networking functionality of a full node but only verifies block headers, without downloading the associated data, or it can a more stripped down client that relies heavily upon forwarding requests to an RPC provider to interact with Ethereum.
+
+## Why are light clients important? {#why-are-light-clients-important}
+
+Light clients matter because they allow users to verify incoming data rather than blindly trusting that their data provider is correct and honest. The data they receive can be checked against block headers that they know have been signed by at least 2/3 of a random set of 512 Ethereum validators. This is very strong evidence that the data is correct, and it doesn't require trusting some specific provider to be honest. The light client requires a tiny amount of computing power, memory and storage so it can be run on a mobile phone, embedded in an app or as part of a browser. Light clients therefore offer trust-minimized access to Ethereum without the need for expensive hardware. Light clients are a way to make trustless access to Ethereum just as frictionless as trusting a third-party provider.
```

Comment:
> Light clients matter because they allow users to verify incoming data rather than blindly trusting that their data provider is correct and honest.

I can see this being the reason for why "nodes" are important... but if we're talking about _light_ clients specifically (topic sentence to the "Why are light clients important?" section) I would make a note about the negligible hardware requirements that this approach offers.

Reply:
```suggestion
Light clients matter because they allow users to verify incoming data rather than blindly trusting that their data provider is correct and honest, while using just a tiny fraction of the computational resources of a full node. The data light clients receive can be checked against block headers that they know have been signed by at least 2/3 of a random set of 512 Ethereum validators. This is very strong evidence that the data is correct. 

The light client only uses a tiny amount of computing power, memory and storage so it can be run on a mobile phone, embedded in an app or as part of a browser. Light clients are a way to make trust-minimized access to Ethereum just as frictionless as trusting a third-party provider.
```